README.md

```diff
@@ -39,7 +39,7 @@ Now we can `npm install` (or `yarn add`) the rest:
 
 ```
 $ cd </path/to/your/lektor/project>/webpack
-$ npm install --save-dev webpack babel-core node-sass babel-loader sass-loader css-loader url-loader style-loader file-loader extract-text-webpack-plugin
+$ npm install --save-dev webpack-cli @babel/core sass babel-loader sass-loader css-loader url-loader file-loader mini-css-extract-plugin
```

Comment:
Is this not missing `webpack` itself?

Reply:
This is coming as a dependency from webpack. I didn't see a direct dependency on webpack, only the cli.
Should it be added anyway? JS is not my mother tongue

Reply:
It might be brought in as a transitive dependency but since we use it directly (for example by requiring it in the config), we should install it as a direct dev dependency.

Reply:
I have removed the direct reference. It is used to add modules that are now included by default.
Sorry, I did mean to answer this earlier.

Reply:
Will add `webpack` in the merge.